### PR TITLE
KCCOV-9 Added tests for components and did some initial cleanup

### PR DESCRIPTION
**Pull request by Misterimpulse**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@
 /public
 /lib
 /target
+.idea
```
Comment (loganom) [inline .gitignore:18]: is the __snapshots__ directory generated? We may want to also ignore it

File: src/components/HomePage.jsx [hunk 2]
```diff
@@ -6,15 +6,22 @@ import { SocialIcon } from 'react-social-icons';
...
+ * The header present in the home page to show off the logo and main title. This is separate from the header that
+ * controls site navigation.
+ */
 const HomeHeader = () => (
   <Row>
     <Col>
-      <img src={logo} style={{ maxWidth: '100%' }} className="header-logo" />
-      <h1>Kansas City's Covid Assistance</h1>
+      <img src={logo} alt="The logo for KC-Covid" style={{ maxWidth: '100%' }} className="header-logo" />
+      <h1>Kansas City `&#39;`s Covid Assistance</h1>
```
Comment (loganom) [inline src/components/HomePage.jsx:17]: May be getting ahead of myself here, but will we want to eventually have a constants file for strings like these?

File: package.json [hunk 3]
```diff
@@ -17,9 +17,9 @@
     "pack": "NODE_ENV=production webpack --config webpack.config.js -p",
     "pack-win": "webpack --config webpack.config.js -p",
     "start": "webpack-dev-server --mode development",
-    "test": "react-scripts test",
+    "test": "npm run test:lint && npm run test:jest",
     "lint:scss": "stylelint src/**/*.scss",
-    "test:lint": "eslint --fix --ext .js,.jsx, . && npm run lint:scss",
+    "test:lint": "eslint --ext .js,.jsx, . && npm run lint:scss",
```
Comment (loganom) [inline package.json:22]: Have you ever used the fix flag in a project before? I'm curious

File: src/App.jsx [hunk 4]
```diff
@@ -1,45 +1,46 @@
...
 import HomePage from './components/HomePage';
 import RequestAidForm from './components/RequestAidForm';
 import VolunteerForm from './components/VolunteerForm';
 
 import './App.scss';
 
-// TODO add logic server side to serve this content correctly with history
-// TODO ErrorBoundary + wrap all components in one
-// TODO INTL
+// TODO KCOVID-13 logic server side to serve this content correctly with history
```
Comment (loganom) [inline src/App.jsx:14]: Does history here refer to the last visited pages by a user or something else?

File: src/components/ErrorBoundary.jsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+  /**
+   * React lifecycle method to update this component state whenever an error is detected.
+   * @param {Object} error The error that was caught.
+   */
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+
+    /* eslint-disable no-console */
+    console.log(`ERROR OCCURRED: ${error}`);
```
Comment (loganom) [inline src/components/ErrorBoundary.jsx:28]: could use console.error if you wanted

File: .gitignore (hunk 1 above)
Comment (Misterimpulse) [reply .gitignore:18]: The snapshot directory should be committed as it's part of the jest tests.

File: src/components/HomePage.jsx (hunk 2 above)
Comment (Misterimpulse) [reply src/components/HomePage.jsx:17]: Yeah ideally we'd have all of these in a translations file, but I think that should be done with the translations story

File: package.json (hunk 3 above)
Comment (Misterimpulse) [reply package.json:22]: Yeah but never in the build, I wouldn't want building the code to change the code

File: src/App.jsx (hunk 4 above)
Comment (Misterimpulse) [reply src/App.jsx:14]: Yeah it's the "history" of the React.Router, so all of the routes you visit are "pushed" onto the history. Even if you navigate there some other way than just calling the push method

File: .gitignore (hunk 1 above)
Comment (loganom) [reply .gitignore:18]: 👍 New to the idea of snapshots here - looks like these are files created by jest to be used for testing? Would we ever modify these?

File: src/components/HomePage.jsx (hunk 2 above)
Comment (loganom) [reply src/components/HomePage.jsx:17]: 👍

File: package.json (hunk 3 above)
Comment (loganom) [reply package.json:22]: No that totally makes sense.. good point.

File: .gitignore (hunk 1 above)
Comment (Misterimpulse) [reply .gitignore:18]: Not manually! Yeah they are autogenerated by the "toMatchScreenshot()" expect

File: src/components/ErrorBoundary.jsx (hunk 5 above)
Comment (Misterimpulse) [reply src/components/ErrorBoundary.jsx:28]: Yup i'll update it!

**Pull request merged**